docs/using/api.md

```diff
@@ -245,7 +245,7 @@ Within the jobs list, a complete job dictionary will look like:
 
 Importantly, the `files` block provides download links for the product files.
 
-For large queries results may be truncated. In this case there will be a `next` key in the response that will contain a url to continue the query (this response may be similarly truncated and include a `next` key).
+For large queries results may be truncated. In this case there will be a `next` key in the response that will contain an url to continue the query (this response may be similarly truncated and include a `next` key).
```

Comment:
Ha, I feel like it could be correct either way depending on how you pronounce "url" ("U.R.L" vs. "earl").

Reply:
Grammatically, the `a` is correct because you're supposed to pronounce it "you-are-ell"

Reply:
but, also, url should be URL

Reply:
@jhkennedy To ensure the spell check passes with "a", we need to remove the indefinite article check in the spell check workflow. I've done this in PR #287

Reply:
@forrestfwilliams pulling it for now is probably the right path, but it might be worth an issue/PR. E.g.,:
https://github.com/retextjs/retext-indefinite-article/issues/16

Reply:
Created `retest-indefinite-article` [PR #19](https://github.com/retextjs/retext-indefinite-article/pull/19) to fix issue